presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -101,10 +103,15 @@ public void testShowCreateView()
     {
         onHive().executeQuery("DROP VIEW IF EXISTS hive_show_view");
 
-        onHive().executeQuery("CREATE VIEW hive_show_view AS SELECT * FROM nation");
```

Comment:
what was wrong about `*`?

can we test both cases, with `*` and with explicit column(s)?

(nation table has few columns, it doesn't hurt to enumerate all of them)